### PR TITLE
Finish Samsung Internet mirroring for api/*

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR finishes the mirroring of Chrome Android to Samsung Internet as a follow-up to #5180, #5181, #5182, and #5183.  This utilizes the mirroring script introduced in #4813, followed by a few manual adjustments.

### CODE REVIEW

File: api/AnimationEvent.json [hunk 1]
```diff
@@ -83,8 +83,8 @@
               "version_added": "4.0"
             },
             {
-              "prefix": "webkit",
-              "version_added": true
+              "version_added": true,
```
Comment (Elchi3) [inline api/AnimationEvent.json]: Why is this change here? It only swaps the order of version_added and prefix?

File: api/Attr.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
           },
           "samsunginternet_android": {
             "version_added": true,
-            "notes": "As of Samsung Internet 5.0, this property no longer inherits from Node."
+            "notes": "As of Samsung Internet 45, this property no longer inherits from Node."
```
Comment (Elchi3) [inline api/Attr.json]: Samsung Internet 45 is not a valid Samsung Internet version.

File: api/AudioParam.json [hunk 3]
```diff
@@ -162,9 +162,9 @@
                 "version_added": "7.0"
               },
               {
-                "alternative_name": "cancelValuesAndHoldAtTime()",
                 "version_added": true,
-                "version_removed": "6.0"
+                "version_removed": "6.0",
```
Comment (Elchi3) [inline api/AudioParam.json]: This again only orders things, right? But not alphabetical?

File: api/CSSKeyframesRule.json [hunk 4]
```diff
@@ -133,9 +133,9 @@
                 "version_added": true
               },
               {
-                "alternative_name": "insertRule",
                 "version_added": true,
-                "version_removed": "5.0"
+                "version_removed": "5.0",
```
Comment (Elchi3) [inline api/CSSKeyframesRule.json]: Ordering only?

File: api/Console.json [hunk 5]
```diff
@@ -343,7 +343,7 @@
               },
               "samsunginternet_android": {
                 "version_added": true,
-                "notes": "In Samsung Internet 1.5, if a negative value is passed to <code>%d</code>, it will be rounded down to the closest negative integer. For example, -0.1 becomes -1."
+                "notes": "In version 28, if a negative value is passed to <code>%d</code>, it will be rounded down to the closest negative integer. For example, -0.1 becomes -1."
```
Comment (Elchi3) [inline api/Console.json]: Samsung Internet 28 is not a valid version.

File: api/Console.json [hunk 6]
```diff
@@ -985,7 +985,7 @@
               },
               "samsunginternet_android": {
                 "version_added": true,
-                "notes": "In Samsung Internet 1.5, if a negative value is passed to %d, it will be rounded down to the closest negative integer, so -0.1 becomes -1."
+                "notes": "In version 28, if a negative value is passed to %d, it will be rounded down to the closest negative integer, so -0.1 becomes -1."
```
Comment (Elchi3) [inline api/Console.json]: version 28 isn't valid

File: api/DeviceOrientationEvent.json [hunk 7]
```diff
@@ -40,7 +40,7 @@
           },
           "samsunginternet_android": {
             "version_added": true,
-            "notes": "Before Samsung Internet 5.0, Samsung Internet provided absolute values instead of relative values for this event. Developers still needing absolute values may use the <code>ondeviceorientationabsolute</code> event."
+            "notes": "Before version 50, Samsung Internet provided absolute values instead of relative values for this event. Developers still needing absolute values may use the <code>ondeviceorientationabsolute</code> event."
```
Comment (Elchi3) [inline api/DeviceOrientationEvent.json]: version 50 is not valid for Samsung

File: api/AnimationEvent.json (hunk 1 above)
Comment (vinyldarkscratch) [reply api/AnimationEvent.json]: My guess is that it's trying to mirror the order of keys from Chrome?  That would explain the various other instances of reordering as well.  (Probably some Node.js thing if I had to guess, I'll certainly have to keep it in mind!)

Comment (Elchi3) [reply api/AnimationEvent.json]: Probably, yeah. I think we should try to have consistent ordering or have the mirroring script ignore this. It creates unnecessary noise when we do script based mass updates.
